tiny_dnn/tiny_dnn.h

```diff
@@ -116,6 +109,26 @@ using ave_unpool = tiny_dnn::average_unpooling_layer;
 
 }  // namespace layers
 
+namespace activation {
+
+using sigmoid = tiny_dnn::sigmoid_layer;
+
```

Comment:
Followed it like layers namespace above, just for the sake of uniformity. Shall I trim it off ?